descriptions/platform.json

```diff
@@ -2,28 +2,35 @@
     "$schema": "https://json-schema.org/draft/2020-12/schema",
     "type": "object",
     "title": "Platform description",
-    "description": "A description of a platform including metadata of the platform itself, available sensors and produced observations",
+    "description": "A description of a platform including metadata of the platform itself and hosted devices",
     "required": [
         "platform_id",
-        "sensors"
+        "devices"
     ],
     "properties": {
         "platform_id": {
             "type": "string",
             "title": "Platform id",
-            "desciption": "Id of the platform"
+            "description": "Id of the platform"
         },
         "metadata": {
             "type": "object",
             "title": "Metadata",
             "description": "Metadata of this platform"
         },
-        "sensors": {
+        "devices": {
             "type": "array",
-            "title": "Sensors",
-            "description": "List of sensors associated with this platform",
+            "title": "Devices",
+            "description": "List of devices associated with this platform",
             "items": {
-                "$ref": "../descriptions/sensor.json"
+                "anyOf": [
+                    {
+                        "$ref": "./devices/generic_device.json"
+                    },
+                    {
+                        "$ref": "./devices/rudder.json"
```

Comment:
Here we should have all the devices and not only `generic_device` and `rudder`. I tried to find in the documentation if there was something like `"anyOf": ["$ref" for device in ./devices]`, but could not find something similar. Perhaps we need to add them by hand.